jenkinsfiles/auth0-users/role_api.py

```diff
@@ -0,0 +1,69 @@
+from role import Role
+
+import requests
+import logging
+
+
+logging.basicConfig()
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
+
+
+class RoleAPI(object):
+
+    def __init__(self, authz_api, authz_token):
+        self.authz_api = authz_api
+        self.authz_token = authz_token
```

Comment:
All these API classes have a lot in common - maybe there should be a superclass?